src/containers/LikeButtons/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { isLoggedIn } from '../../utils/auth';
+import {
+  liker, disliker, getLikes,
+} from '../../redux/actions';
+
+
+class LikeButton extends Component {
+  componentDidMount = () => {
+    const { slug } = this.props;
+    const { fetchLikes } = this.props;
+
+    fetchLikes({
+      slug,
+    });
+  }
+
+  handleLikeClick = () => {
+    const { addLike } = this.props;
+    const { slug } = this.props;
+
+    addLike({
+      slug,
+    });
+  }
+
+  handleDislikeClick = () => {
+    const { addDislike } = this.props;
+    const { slug } = this.props;
+
+    addDislike({
+      slug,
+    });
+  }
+
+  render() {
+    const {
+      liked, disliked, likeCount, dislikeCount
+    } = this.props;
+    const LikeLabel = liked ? 'Unlike' : 'Like';
+    const DislikeLabel = disliked ? 'Undislike' : 'Dislike';
+    return (
+      <div className="customContainer">
```

Comment:
all the jsx should be in a dumb component, then render the component here.

Reply:
On it.